### ISSUE TITLE
为什么我的test loss一直降不下去？？？很高，mAP值在40%左右

### ISSUE THREAD

**Issue by wanzghen0123** [YunYang1994/tensorflow-yolov3#303]
训练类别数20，数据量3000，批大小12 。训练了300多个epoch，loss值一直在10～20之间．
train loss: 11.86: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████| 209/209 [01:48<00:00,  2.02it/s]
=> Epoch: 314 Time: 2019-08-31 08:29:37 Train loss: 14.18 Test loss: 17.18 Saving ./checkpoint/yolov3_test_loss=17.1838.ckpt ...
train loss: 9.13: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████| 209/209 [01:47<00:00,  1.74it/s]
=> Epoch: 315 Time: 2019-08-31 08:31:42 Train loss: 14.08 Test loss: 17.21 Saving ./checkpoint/yolov3_test_loss=17.2069.ckpt ...
train loss: 11.57: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████| 209/209 [01:50<00:00,  2.02it/s]
=> Epoch: 316 Time: 2019-08-31 08:33:50 Train loss: 14.03 Test loss: 17.30 Saving ./checkpoint/yolov3_test_loss=17.2983.ckpt ...
train loss: 16.18: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████| 209/209 [01:50<00:00,  1.92it/s]
=> Epoch: 317 Time: 2019-08-31 08:35:58 Train loss: 14.05 Test loss: 16.99 Saving ./checkpoint/yolov3_test_loss=16.9910.ckpt ...
train loss: 18.37: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████| 209/209 [01:52<00:00,  2.33it/s]
=> Epoch: 318 Time: 2019-08-31 08:38:08 Train loss: 13.84 Test loss: 17.47 Saving ./checkpoint/yolov3_test_loss=17.4713.ckpt ...

**Comment by wanzghen0123** [comment]
自己制作的voc类型的数据集，配置参数如下：
__C.TRAIN.ANNOT_PATH            = "./data/dataset/voc_train.txt"
__C.TRAIN.BATCH_SIZE            = 14
__C.TRAIN.INPUT_SIZE            = [320, 352, 384, 416, 448, 480, 512, 544, 576, 608]
__C.TRAIN.DATA_AUG              = True
__C.TRAIN.LEARN_RATE_INIT       = 1e-3
__C.TRAIN.LEARN_RATE_END        = 1e-3
__C.TRAIN.WARMUP_EPOCHS         = 2
__C.TRAIN.FISRT_STAGE_EPOCHS    = 400 #20
__C.TRAIN.SECOND_STAGE_EPOCHS   = 500 #30
__C.TRAIN.INITIAL_WEIGHT        = "./checkpoint/yolov3_test_loss=10.4744.ckpt-327"

－－－－－－－－－－－－－－－－－－－－－－
麻烦帮忙看一下，谢谢

**Comment by myrzx** [comment]
你好，请问现在你解决了吗

**Comment by yanchangqin** [comment]
我也是一直在10-20之间

**Comment by zhishao** [comment]
学习率太高了吧 降到1e-5/6可能好点吧
